pkg/kubelet/config/common.go

```diff
@@ -73,6 +73,20 @@ func applyDefaults(pod *api.Pod, source string, isFile bool, nodeName types.Node
 	// The generated UID is the hash of the file.
 	pod.Annotations[kubetypes.ConfigHashAnnotationKey] = string(pod.UID)
 
+	if isFile {
+		// Applying the default Taint tolerations to static pods
+		pod.Spec.Tolerations = append(pod.Spec.Tolerations, api.Toleration{
+			Key:      "node.alpha.kubernetes.io/unreachable",
```

Comment:
ok, I didn't know if validation would complain if the same toleration showed up twice